src/main/resources/avro/bdg.avdl

```diff
@@ -363,12 +363,55 @@ record NucleotideContigFragment {
  duplicated sequence.
  */
 enum StructuralVariantType {
+
+  /**
+   Breakend.  VCF INFO reserved key "SVTYPE" value "BND".
+   */
+  BREAKEND,
+
+  /**
+   Copy number variable region (may be both deletion and duplication).
+   VCF INFO reserved key "SVTYPE" value "CNV".
+   */
+  COPY_NUMBER_VARIABLE,
```

Comment:
`s/variable/variant/g`? Or, is `variable` meant to indicate that the copy number of this region varies?